packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -389,18 +403,21 @@ tar -xz -C "$DD_HOME/agent" --strip-components 1 -f "$DD_HOME/agent.tar.gz"
 rm -f "$DD_HOME/agent.tar.gz"
 print_done
 
+# get the version from the actual config file in the branch
+AGENT_VERSION=$(cat $DD_HOME/agent/config.py | grep AGENT_VERSION | cut -f2 -d'=')
```

Comment:
One more thing: let's use a different variable name than `AGENT_VERSION` to avoid overwriting the existing var :)